package.json

```diff
@@ -45,17 +45,16 @@
     "trailingComma": "es5"
   },
   "devDependencies": {
+    "@babel/core": "^7.0.0",
```

Comment:
Here's some wildcarded dependencies again. Maybe ok for a dev dependency?

Reply:
TBH I'm kind of torn on this though. With lockfiles in place, there's no real reason to worry about these unless we decide to adopt an automated dependency updating system like Renovate. The package-lock will ensure that all versions stay the same until explicitly changed... It seems like an unnecessary fight against the default `npm install -D` behavior at this point.

I guess some people like to refer to the package.json for exact versions rather than doing `npm list <package>` ?